### ISSUE TITLE
Spyder has stopped working

### ISSUE THREAD

**Issue by mattmecca** [spyder-ide/spyder#6201]
## Description

**What steps will reproduce the problem?**

1. Trying to run any line of code
2. 
3. 

**What is the expected output? What do you see instead?**

[1]: import numpy as np

"Spyder has encountered a problem.
Sorry for the inconvenience."

**Please provide any additional information below**

  File "/Users/MattMecca/anaconda3/lib/python3.6/site-packages/spyder/app/mainwindow.py", line 2463, in execute_in_external_console
    console.execute_code(lines)
  File "/Users/MattMecca/anaconda3/lib/python3.6/site-packages/spyder/plugins/ipythonconsole.py", line 954, in execute_code
    sw.execute(to_text_string(lines))
  File "/Users/MattMecca/anaconda3/lib/python3.6/site-packages/qtconsole/history_console_widget.py", line 47, in execute
    source, hidden, interactive)
  File "/Users/MattMecca/anaconda3/lib/python3.6/site-packages/qtconsole/console_widget.py", line 632, in execute
    self._execute(source, hidden)
  File "/Users/MattMecca/anaconda3/lib/python3.6/site-packages/qtconsole/frontend_widget.py", line 294, in _execute
    msg_id = self.kernel_client.execute(source, hidden)
AttributeError: 'NoneType' object has no attribute 'execute'

## Version and main components

* Spyder Version: 3.2.4 
* Python Version: 3.6.3
* Qt Versions: 5.6.2, PyQt5 5.6 on Darwin

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.20.3 (OK)
numpy >=1.7      :  1.13.3 (OK)
sphinx >=0.6.6   :  1.6.3 (OK)
rope >=0.9.4     :  0.10.5 (OK)
jedi >=0.9.0     :  0.10.2 (OK)
psutil >=0.3     :  5.4.0 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
sympy >=0.7.3    :  1.1.1 (OK)
cython >=0.21    :  0.26.1 (OK)
qtconsole >=4.2.0:  4.3.1 (OK)
IPython >=4.0    :  6.1.0 (OK)
pylint >=0.25    :  1.7.4 (OK)

```

**Comment by CAM-Gerlach** [comment]
Thanks for reporting. This error appears similar to #6035 , which was fixed by the user resetting preferences with ``spyder --reset`` at the command line. Please try that, along with the other steps in our [Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ), specifically the [Basic First Aid](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ#basic-first-aid-general-troubleshooting) and [Emergency CPR](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ#emergency-cpr-spyder-wont-launch) sections. If you still can't get it to work, please follow the procedure in the [Calling for Help](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ#calling-for-help-still-have-a-problem) section, including the requested information in a followup comment here. Otherwise, if we don't hear from you, this issue will be closed after one week (7 days).

**Comment by mattmecca** [comment]
Looks like it worked. Thanks, Matt.

**Comment by CAM-Gerlach** [comment]
Closed as duplicate of #6035
